app/uk/gov/hmrc/helptosavefrontend/views/register/create_account.scala.html

```diff
@@ -40,119 +42,159 @@
-      (Messages("hts.register.create_account.change") + " " + Messages("hts.email-saving-remainders.title.h1")),
-      "change-reminder"
-    ),true
-  )
-    }
+    @if(period.matches("none")) {
+    @viewHelpers.govukSummaryList(SummaryList(
+    rows = Seq(SummaryListRow(
+      key = Key(content = Text(Messages("hts.email-saving-remainders.title.h1"))),
+          value = Value(content = Text(Messages("hts.register.create_account.your-remainder.note"))),
+          actions = Some(Actions(
+          items = Seq(ActionItem(
+          href = routes.RegisterController.changeEmail.url,
+          content = Text(Messages("hts.register.create_account.change"))
+          ))
+        ))
+    ))))
+    } else{
+        @viewHelpers.govukSummaryList(SummaryList(
+        rows = Seq(SummaryListRow(
+        key = Key(content = Text(Messages("hts.email-saving-remainders.title.h1"))),
+            value = Value(content = Text(Messages("hts.reminder-confirmation.title.p1-1") +" "+
+                  PeriodUtils.getMessage(period)
+                  +" "+ Messages("hts.reminder-confirmation.title.p1-2"))),
+            actions = Some(Actions(
+            items = Seq(ActionItem(
+              href = routes.RegisterController.changeEmail.url,
+              content = Text(Messages("hts.register.create_account.change"))
+              ))
+            ))
+        ))))
       }
+    }
   </dl>
 
-  @summary_section_head(heading = Messages("hts.register.create_account.your-bank-details"))
+<h2 class="govuk-heading-m">@messages("hts.register.create_account.your-bank-details")</h2>
   <dl class="govuk-check-your-answers cya-questions-long">
-    @summary_row(Messages("hts.register.create_account.your-bank-details.sort-code"), display2CharFormat(bankDetails.sortCode.toString()),
-      summary_section_change(
-        (routes.RegisterController.changeBankDetails.url + "#sortCode"),
-        Messages("hts.register.create_account.change-your-sort-code"),
-        "change-your-sort-code"
-      )
-    )
-    @summary_row(Messages("hts.register.create_account.your-bank-details.account-number"), bankDetails.accountNumber,
-      summary_section_change(
-        (routes.RegisterController.changeBankDetails.url + "#accountNumber"),
-        Messages("hts.register.create_account.change-your-account-number"),
-        "change-your-account-number"
-      )
-    )
+      @viewHelpers.govukSummaryList(SummaryList(
+      rows = Seq(SummaryListRow(
+      key = Key(content = Text(Messages("hts.register.create_account.your-bank-details.sort-code"))),
+      value = Value(content = Text(Messages(display2CharFormat(bankDetails.sortCode.toString())))),
+      actions = Some(Actions(
+      items = Seq(ActionItem(
+      href = routes.RegisterController.changeBankDetails.url + "#sortCode",
+      content = Text(Messages("hts.register.create_account.change"))
+      ))
+      ))
+      ))))
+
+      @viewHelpers.govukSummaryList(SummaryList(
```

Comment:
should not be a new summary list for each row! the summary list above should have a Sequence (which is a comma separated list) of SummaryListRows, in order to build the list given there are conditions for including some rows it might be better to have a separate method to build the rows and then feed in that as a single value